pkg/cloud/aws/services/ec2/instances.go

```diff
@@ -437,13 +439,20 @@ func (s *Service) runInstance(role string, i *v1alpha1.Instance) (*v1alpha1.Inst
 		return nil, errors.Errorf("no instance returned for reservation %v", out.GoString())
 	}
 
-	s.scope.V(2).Info("Waiting for instance to run", "instance-id", *out.Instances[0].InstanceId)
-	err = s.scope.EC2.WaitUntilInstanceRunningWithContext(
-		aws.BackgroundContext(),
+	waitTimeout := 1 * time.Minute
+	s.scope.V(2).Info("Waiting for instance to be in running state", "instance-id", *out.Instances[0].InstanceId, "timeout", waitTimeout.String())
+	ctx, cancel := context.WithTimeout(aws.BackgroundContext(), waitTimeout)
```

Comment:
any particular reason to use aws.BackgroundContext() here? It looks like it just returns context.Background().

Reply:
Mostly because it's there, I can switch if necessary :D